src/index.js

```diff
@@ -0,0 +1,177 @@
+    const weatherLandscape = document.getElementById("sky");
+
+    if (skySelectElement.value === "sunny") {
+        weatherLandscape.className = "skySunny";
+        console.log("SUNNY")
+    } else if (skySelectElement.value === "partlyCloudy") {
+        weatherLandscape.className = "skyPartlyCloudy";
+    } else if (skySelectElement.value === "overcast") {
+        weatherLandscape.className = "skyOvercast";
+    } else if (skySelectElement.value === "rainy") {
+        weatherLandscape.className = "skyRainy";
+    } else if (skySelectElement.value === "snow"){
+        weatherLandscape.className = "skySnow";
+    };
+};
+// create a skyToggle function that changes the images of the sky
+// it will be almost the same as tempTheme, getting element id #sky
+// and then if one of the buttons from select_sky function is called
+// if that list element (doc.getElementbyId) sky-id 
+// is equal to rain change className to skyRainy
+// if that list elelment sky-id is equal to snow
+// change className to skySnow
+// if sky-id is equal to partyCloudy, change classname to skyPartlyCloudy
+// if sky-id is equal to sunny, change className to skySunny
+
+
+const cityNameResponse = () => {
+
+    const inputCity = document.getElementById('cityText').value;
+    const cityDisplay = document.getElementById('city');
+    state.city = inputCity
+    cityDisplay.textContent = state.city;
+};
+
+const getLatAndLong = () => {
+    axios
+    .get('http://localhost:5000/location',{
+    params: {
+    q: state.city,
+    },
+    }).then((response) => {
+    state.lat = response.data[0]['lat'];
+    state.lon = response.data[0]['lon'];
+    getWeather();
+    })
+    .catch(err => console.log('error finding latitude and longitude', {err}));
+};
+
+const getIRLTemp = () => {
+    state.city = document.getElementById("cityText").value;
+    getLatAndLong();
+};
+
+const tempKtoC = (temp) => {
+    return (temp - 273.15);
+};
+
+const tempKtoF = (temp) => {
+    return (temp-273.15) * (9/5) + 32;
+}
+
+const getWeather = () => {
+    axios
+    .get(`http://localhost:5000/weather`,{
+    params: {
+        lat: state.lat,
+        lon: state.lon,
+    },
+    }).then((response) => {
+        const weather = response.data;
+        console.log(weather)
+        state.temp = Math.round(tempKtoC(weather.current.temp));
+        tempThemeToggle();
+        // add validate weather call here
+    })
+    .catch(err => console.log('error finding weather data', {err}));
+};
+
+// const show_sky = () => {
+//     let weather = document.getElementById("sky-id");
+    
+//     if (weather.style.display == "block") {
+//         weather.style.display == "none";
+//     } else {
+//         weather.style.display = "block";
+//     };
+// };
+
+// window.onclick = function (event) {
+//     if (!event.target.matches('.dropdown_button')) {
+//         document.getElementById('sky-id')
+//         .style.display = "none";
+//     };
+// }
```

Comment:
I can't remember if we touched on this in a meeting, but it's nice to see your found a solution for the skies without using the `style` attribute. Setting the `style` attribute directly on an element, is the javascript equivalent of adding inline CSS styles, which is not considered best practice.